src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
What is the reason for that? I thought if we have them in chameleon we get them from there and then merge with what we have in this file (so the rest of containers unsupported by chameleon), and then this is what we give as options to the user. What are we really mapping here?